src/storage/mvcc/reader.rs

```diff
@@ -273,6 +273,48 @@ impl<'a> MvccReader<'a> {
         Ok(())
     }
 
+    pub fn seek_first(&mut self, ts: u64) -> Result<Option<Key>> {
+        assert!(self.scan_mode.is_some());
+        self.create_write_cursor()?;
+        self.create_lock_cursor()?;
+
+        let mut w_cur = self.write_cursor.as_mut().unwrap();
+        let mut l_cur = self.lock_cursor.as_mut().unwrap();
```

Comment:
I think we can speed up by  scanning the write column family only, since the primary's lock (which we are looking for) would not leave a lock for a long time. @disksing what's your opinion？